squid/conf/squid.conf

```diff
@@ -0,0 +1,55 @@
+http_port 3128
+
+maximum_object_size 100 MB
+max_stale 2 weeks
+
+refresh_pattern . 60 50% 14400 store-stale
```

Comment:
Can you guys find some docs that explain what this means? Can add a separate section to bottom of README about configuring squid, if we need.